pycqed/measurement/calibration_toolbox.py

```diff
@@ -75,6 +75,7 @@ def mixer_carrier_cancellation(SH, source, MC,
                     'sigma0':1,
                     'options': {'maxiter': 300,    # maximum function cals
                                 # Scaling for individual sigma's
+                                'ftarget': -110,
```

Comment:
Does not belong in this PR but a good change anyway. Maybe add a comment why this is the right target for optimization.

Reply:
Comment added: -110 is the noise level of the SH

Reply:
Conflict is in this file: will delete from this commit, and update in Develop after merging